internal/pkg/describe/lb_web_service.go

```diff
@@ -289,36 +291,76 @@ type WebServiceRoute struct {
 	URL         string `json:"url"`
 }
 
-// ServiceDiscovery contains serialized service discovery info for an service.
-type ServiceDiscovery struct {
+type internalEndpoint struct {
 	Environment []string `json:"environment"`
-	Namespace   string   `json:"namespace"`
+	Endpoint    string   `json:"endpoint"`
 }
 
-type serviceDiscoveries []*ServiceDiscovery
+type internalEndpoints []*internalEndpoint
```

Comment:
i think this could be `[]internalEndpoint` if L318 was changed to:
```
for i := range ies {
  if ies[i].Endpoint == endpoint {
    ies[i].Environment = append(ies[i].Environment, env)
  }
}
```

Reply:
Discussed offline and it ended up if we use `[]internalEndpoint` then it'll be `(*ies)[i]` to refer to struct. Using `[]* internalEndpoint` is probably more readable here.

Reply:
Could `internalEndpoints` be a `map[Endpoint]Environments` where `Endpoint string` and `Environments []stirng`? Would this increase readability?

Reply:
I like this idea. i think readability is about the same but it'll be more efficient